### PR TITLE
Api review clean up

### PR DESCRIPTION
**Pull request by simplynaveen20**
This PR is doing clean up for API review for it GA plan.
No api surface change, no logical change

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java [hunk 1]
```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.encryption.keyprovider;
+package com.azure.comsos.encryption.implementation.keyprovider;
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java:4]: should be cosmos?

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
 
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.EncryptionKeyStoreProvider;
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.KeyEncryptionKeyAlgorithm;
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.MicrosoftDataEncryptionException;
+import com.azure.cosmos.encryption.keyprovider.EncryptionKeyWrapProvider;
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java:10]: nit: sort by order?

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java (hunk 1 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java:4]: Thanks for catching this

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java (hunk 2 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java:10]: Imports are already sorted , if that you meant ?

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Review by aayush3011 — APPROVED** [review]
LGTM, thanks

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @simplynaveen20

**Pull request merged**